### ISSUE TITLE
CST463 Exam Issues notes

### ISSUE THREAD

**Issue by samogden** [OtterDen-Lab/QuizGenerator#39]
This is a list of issues I noticed while grading the CST463 exam

**Comment by samogden** [comment]
- [ ] Matrix Multiplication always works
- [ ] Set limits on how big matrices and vectors can be
- [ ] Make sure that gradient descent converges
  - [ ] decrease learning rate
  - [ ] Come up with a new way of generating equations, because none of these converge and they all have 
- [ ] the e^x gradient problems must go.
- [ ] For the MSE questions, generate an actual line and then pick data nearby it.  Like make it more reasonable.
- [ ] For logloss, make sure there's at least one really close prediction, like within 0.95 for true
